core/invehicle-digital-twin/src/main.rs

```diff
@@ -17,42 +15,41 @@ use std::net::SocketAddr;
 use std::sync::Arc;
 use tonic::transport::Server;
 use tonic::{Request, Status};
-use url::Url;
 
-mod digitaltwin_impl;
 mod invehicle_digital_twin_config;
-mod providerservice_impl;
+mod invehicle_digital_twin_impl;
 
-pub const DIGITAL_TWIN_SERVICE_NAME: &str = "digital_twin";
-pub const DIGITAL_TWIN_SERVICE_VERSION: &str = "1.0";
+pub const INVEHICLE_DIGITAL_TWIN_SERVICE_NAME: &str = "invehicle_digital_twin";
+pub const INVEHICLE_DIGITAL_TWIN_SERVICE_VERSION: &str = "1.0";
 pub const CHARIOTT_NAMESPACE_FOR_IBEJI: &str = "sdv.ibeji";
+pub const CHARIOTT_COMMUNICATION_KIND_FOR_GRPC: &str = "grpc+proto";
+pub const CHARIOTT_COMMUNICATION_REFERENCE_FOR_INVEHICLE_DIGITAL_TWIN_SERVICE: &str =
+    "invehicle_digital_twin.v1";
```

Comment:
In Chariott and Agemo communication_kind contains the extension at the end, i.e. ".proto", but this can be up to the consumer for how they want to use this field

Reply:
I'm going to use a URI.